frontend/src/components/core/Block/Block.tsx

```diff
@@ -239,7 +208,7 @@ class Block extends PureComponent<Props> {
       return (
         <StyledColumn
           key={index}
-          className="stBlock"
+          data-testid="stBlock"
```

Comment:
Stupid question: what's the motivation for switching to "data-testid" instead of "class"? (aka I realized I have no idea what "data-testid" even does...)

Reply:
Essentially, classes are no longer used, and it's a more common practice to use a data attribute to give more meaning to the element itself rather than class (which is used for targeting styles).

I chose `testid` because it's used in [React Testing Library](https://testing-library.com/docs/dom-testing-library/api-queries/#bytestid) (Second popular to Enzyme that even [React prefers](https://reactjs.org/docs/test-utils.html#overview)). We don't use it, but it's just future proofing in case we want to move it there.

Reply:
Also: Not stupid at all a question! 😊